src/ec/packages/ec-component-language-list/language-list-splash.html.twig

```diff
@@ -0,0 +1,66 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "items" (array) (default: [])
+    - "lang" (string) (default: '')
+    - "label" (string) (default: '')
+    - "path" (string) (default: '')
+    - "active (boolean) (default: false)
+  - "overlay" (boolean) (default: false): is the list an overlay?
+  - "icon_path" (string) (default: ''): file containing the svg icons
+  - "logo" (object) (default: {}): EC logo (splash only)
+    - "alt" (string) (default: ''): alt text for the logo
+    - "path" (string) (default: ''): path to the logo svg
+  - "close_label (string) (default: ''): label of close button (overlay only)
+  - "title" (string) (default: ''): title of the page (overlay only)
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _items = items|default([]) %}
+{% set _overlay = overlay|default(false) %}
+{% set _icon_path = icon_path|default('') %}
+{% set _logo = logo|default({}) %}
+{% set _close_label = close_label|default('') %}
+{% set _title = title|default('') %}
+
+{% set _css_class = 'ecl-language-list ecl-language-list--splash' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{% set _column1 = _items|slice(0, (items.length / 2)|round(0, 'ceil')) %}
+{% set _column2 = _items|slice((items.length / 2)|round(0, 'ceil'), items.length) %}
+
+{# Print the result #}
+
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <header class="ecl-language-list__header">
+    <img class="ecl-language-list__logo" src="{{ _logo.path }}" alt="{{ _logo.alt }}" />
+  </header>
+
+  <div class="ecl-language-list__container ecl-container">
+    {% include './language-list.html.twig' with {
+      items: _items,
+      overlay: _overlay,
```

Comment:
Now that splash/overlay are split and react implementation not dealing with this property, I think it can be removed as well.